test/EFCore.PG.FunctionalTests/Query/NorthwindMiscellaneousQueryNpgsqlTest.cs

```diff
@@ -356,12 +205,13 @@ public async Task Array_All_ILike(bool isAsync)
         }
 
         #endregion Any/All Like
-
-        #region Helpers
+        protected override void ClearLog()
+            => Fixture.TestSqlLoggerFactory.Clear();
```

Comment:
This is because TestSqlLoggerFactory is a relational-only thing (SQL), but the test base class isn't relational - it is used also for non-relational databases. So you can think about ClearLog as a general, "clear all logs" that isn't SQL specific, which providers must then override to clear the log in their particular way (for relational databases that means clearing SQL).

For test base classes which are relational only this can and should be implemented at the base class.